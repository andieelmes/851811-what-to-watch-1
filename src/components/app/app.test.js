import React from 'react';
import renderer from 'react-test-renderer';
import App from 'components/app/app.jsx';

it(`App component renders correctly`, () => {
  const tree = renderer
    .create(<App/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
