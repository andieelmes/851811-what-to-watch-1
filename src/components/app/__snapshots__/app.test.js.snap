// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render correctly for guests 1`] = `null`;

exports[`App component should render correctly when authorized 1`] = `null`;
